### PR TITLE
APP-478: [C] Include header for using send_event function

### PR DESCRIPTION
**Pull request by Dubland**
/cc @vadimol @Slash32

### CODE REVIEW

File: client/client-multi/client-c/src/extensions/event/kaa_event.h [hunk 1]
```diff
@@ -101,6 +101,25 @@ kaa_error_t kaa_event_remove_transaction(kaa_event_manager_t *self, kaa_event_bl
 
 const char *kaa_find_class_family_name(const char *fqn);
 
+/*
+ * @brief Sends raw event
+ *
+ * It is not recommended to use this function directly. Instead you should use
```
Comment (ohamov) [inline client/client-multi/client-c/src/extensions/event/kaa_event.h]: What about this comment?

Comment (Dubland) [reply client/client-multi/client-c/src/extensions/event/kaa_event.h]: We do not use this function directly, but it is called in generated files.

Comment (vadimol) [reply client/client-multi/client-c/src/extensions/event/kaa_event.h]: This looks odd.

**Review by ohamov — CHANGES_REQUESTED** [review]
That's not the right solution. As the comment states, the function should not be used directly and it is not a part of the interface. The function declaration resides in `kaa_event_private.h`. Just add the necessary include to `kaa_event_families.cvm`.

**Review by vadimol — APPROVED** [review]

**Review by ohamov — APPROVED** [review]

**Comment by vadimol** [comment]
Jenkins failure is not related.

**Pull request merged**